packages/core/src/integrations/supabase.ts

```diff
@@ -448,6 +454,7 @@ function instrumentPostgRESTFilterBuilder(PostgRESTFilterBuilder: PostgRESTFilte
                   return res;
                 },
                 (err: Error) => {
+                  // TODO: shouldn't we capture this error?
```

Comment:
@onurtemizkan not sure if this is me missing something or if we should call `captureException` here. Can you take a look?

Reply:
Hmm, looks like this comes from the original implementation we ported: 

https://github.com/supabase-community/sentry-integration-js/blob/9fc7be61d2890f87257c4af528271374c89c4acd/v8.js#L237-L242

I don't see a reason why we should not capture this, though.

Reply:
thanks! Probably just an oversight then. I'll make a separate PR adding a captureException call here. Thx!